apps/web/app/api/auth/[...nextauth]/auth.ts

```
@@ -5,153 +5,167 @@ import CredentialsProvider from "next-auth/providers/credentials";
+          },
+        });
+        if (!user) {
+          return null;
+        }
+        const isJustGoogleUser = !user.salt;
+        if (isJustGoogleUser) {
+          return null;
+        }
+        const hashedPassword = await bcrypt.hash(
+          credentials.password,
+          user.salt!
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)